subsys/net/openthread/rpc/common/ot_rpc_types.h

```diff
@@ -28,3 +31,6 @@ enum ot_rpc_link_mode_offsets {
 };
 
 typedef uint32_t ot_msg_key;
+typedef uint32_t ot_socket_key;
```

Comment:
ot_msg_key, ot_socket_key - Would it be beneficial to document usage scenarios of above types?